src/chunkserver/filesystem_adaptor/curve_filesystem_adaptor.cpp

```diff
@@ -176,14 +160,108 @@ bool CurveFilesystemAdaptor::rename(const std::string& old_path,
                                             const std::string& new_path) {
     if (!NeedFilter(new_path) && lfs_->FileExists(new_path)) {
         // chunkfilepool内部会检查path对应文件合法性，如果不符合就直接删除
-        chunkFilePool_->RecycleFile(new_path);
+        filePool_->RecycleFile(new_path);
     }
     return lfs_->Rename(old_path, new_path) == 0;
 }
 
+bool CurveFilesystemAdaptor::link(const std::string& old_path,
```

Comment:
Why was this part of the code moved out of the sanpshot directory?

Reply:
becase curve segment also should use CurveFilesystemAdaptor to access localfilesysterm.